README.md

```diff
@@ -44,6 +44,55 @@ resource "doppler_secret" "db_password" {
 }
 ```
 
+## Referencing Secrets Using Multiple Access Tokens
+
+```
+terraform {
+  required_providers {
+    doppler = {
+      source = "DopplerHQ/doppler"
+      version = "1.0.0"
+    }
+  }
+}
+
+variable "doppler_token_dev" {
+  type = string
+  description = "A token to authenticate with Doppler"
```

Comment:
Should we make these descriptions different? Potentially including the environment

Reply:
Yeah, I can do that. It occurred to me, but I figured the name of the variable would be changing depending upon project. However, for this example it would make sense for sure.